### DISCUSSION TITLE
pg_dump: [archiver (db)] query failed: ERROR:  relation "public.build_event_id_seq_XXXXXXX" does not exist

### DISCUSSION THREAD

**Discussion by arpitvipulsharma** [concourse/concourse#6969] [Help & Support]
Hello Team,
We are facing an issue with the backup of Postgres for the concourse. We get error during backup as 
pg_dump: [archiver (db)] query failed: ERROR:  relation "public.build_event_id_seq_4327391" does not exist
Also, it occurs sometimes, not always. 
Concourse version: v7.1.0
Postgres version: PostgreSQL 10.4
Can you help us with it?
Thanks

**Comment by msenmurugan** [comment]
@scottietremendous We are also facing the same issue during backup of Postgres. And it's an issue rather than Q&A. Can you please help us in fixing this issue?

@arpitvipulsharma Are you able to fix this issue?

**Reply by arpitvipulsharma** [reply]
@msenmurugan No, We are still facing this issue. 
@scottietremendous Hi, can you help here?

**Reply by snps-alvinlam** [reply]
A potential fix may be in 7.3.0
https://github.com/concourse/concourse/issues/6588
https://github.com/concourse/concourse/milestone/74?closed=1

**Comment by taylorsilva** [comment]
Depending on how busy your Concourse is, it's very likely you'll run into this issue. Concourse is creating and deleting a lot of these `build_event` tables all the time. What happens in this case is that when the backup started it saw a build_events table, but by the time it went to read and backup the table, it's gone. Thus pg_dump errors.

The solution, which is what we implemented in the BBR PR (https://github.com/concourse/concourse-bosh-release/pull/145) is to simply re-try the backup.
